src/ngraph/op/non_max_suppression.cpp

```diff
@@ -30,10 +30,12 @@ op::v1::NonMaxSuppression::NonMaxSuppression(
     const Output<Node>& iou_threshold,
     const Output<Node>& score_threshold,
     const op::v1::NonMaxSuppression::BoxEncodingType box_encoding,
-    const bool sort_result_descending)
+    const bool sort_result_descending,
```

Comment:
Why do you modify the existing operation? We need to **add** new version of the operation! I recommend to use inheritance since the core functionality of the operation will be re-used.

Reply:
My opinion is that a brand new operation is overkill in this case.  The simple addition of a default parameter for `output_type` to the v1 op should suffice.

Reply:
It needs to be a new op. Any backend already implementing the old op will not know about the new attribute and do the wrong thing.